OCR_pollen.qmd

```diff
@@ -7,17 +7,78 @@ output:
   html_document:
    toc: true
    toc_float: true
----
+
 ```{=html}
 <style>
 body {
 text-align: justify}
 </style>
 ```
 
-# PARTIE 3: GRAZING EFFECT ON POLLINISATION STRATEGIES 
-packages
-```{r pollen_read_data}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+                      INTRODUCTION
```

Comment:
please use markdown formatting using # for section title and  ## for subsection and so on 
this is not a script file with comments.